style.css

```diff
@@ -0,0 +1,175 @@
+body {
+  width: 100%;
+}
+
+/* Links */
+
+body a {
+  text-decoration: none;
+}
+
+.main-footer a {
+  color: #08c;
+}
+
+.right-side-footer-container a {
+  border-right: 1px solid gray;
+  padding: 0 10px;
+}
+
+.showcase-content a {
+  color: white;
+  font-size: 25px;
+  margin-top: 35px;
+  font-family: "Lato", sans-serif;
+  font-weight: 100;
+}
+
+header .nav-bar-ul li a {
+  font: 12px/18px "Lucida Grande", "Lucida Sans Unicode", Helvetica, Arial, Verdana, sans-serif;
+  color: #fff;
+}
+
+a.no-border {
+  border-right: #fff;
+}
+
+.showcase-content a:hover {
+  text-decoration: underline;
+}
+
+header .nav-bar-ul li a:hover {
+  text-decoration: none;
+}
+
+/* Showcase (header) Section */
+
+.header-section-container {
+  max-width: 1430px;
+  margin: 0 auto;
+}
+
+.showcase {
+  background: url(./img/main-background.jpg) no-repeat center center/cover;
+  background-position: 0 92%;
+}
+
+header {
+  height: 600px;
+}
+
+header .showcase-content {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  color: #fff;
+}
+
+header .nav-bar-ul {
+  display: flex;
+  width: auto;
+  margin-top: 90px;
+}
+
+header .nav-bar-ul li {
+  width: 105px;
+  height: auto;
+  padding: 8px;
+  background: linear-gradient(180deg, rgb(75, 73, 73) 50%, rgba(133, 133, 133, 1) 82%);
+  cursor: pointer;
+  border: transparent;
+  border-left: 1px solid gray;
+  text-align: center;
+}
+
+header .nav-bar-ul li:hover {
+  background: rgb(41, 40, 40);
+}
+
+header .apple-link {
+  border-top-left-radius: 5px;
+  border-bottom-left-radius: 5px;
+}
+
+header .apple-link i {
+  font-size: 18px;
+}
+
+.header-search-form {
+  position: relative;
+  font: 12px/18px "Lucida Grande", "Lucida Sans Unicode", Helvetica, Arial, Verdana, sans-serif;
+  color: #fff;
+  display: flex;
+  justify-content: center;
```

Comment:
## Using the Same Rule Repetitively[1]
- [ ] Create a class once and the use in each HTML tag where you need it.
- [ ] Delete all your `color: #fff;` properties in the entire CSS file
- [ ] Add your HTML tags

**Example:**

**CSS file**

```
.white-color{
  color: #fff;
}
```
**Html file**

```
<div class="white-color">
```
**Note:** Do this change to all properties in this file  also that you have like `color: white`

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_

Reply:
Deleted all color: #fff; properties and summarized them in one class